{"uuid": "9773e945-3a99-4419-a493-0554a2dfd6d8", "children": ["c6e5e7cc-eb0b-448a-806e-c2294dd15f6b"], "befores": [{"name": "_xunit_setup_method_fixture_TestRegistration", "status": "passed", "start": 1696247287527, "stop": 1696247316108}], "afters": [{"name": "_xunit_setup_method_fixture_TestRegistration::0", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: HTTPSConnectionPool(host='gauto2.forstudy.spase', port=443): Max retries exceeded with url: /api/users (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x0000022F70292890>: Failed to resolve 'gauto2.forstudy.spase' ([Errno 11001] getaddrinfo failed)\"))\n", "trace": "  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\_pytest\\python.py\", line 910, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\_pytest\\python.py\", line 777, in _call_with_optional_argument\n    func(arg)\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\test_Homework_12.py\", line 30, in teardown_method\n    self._session.delete(url=\"https://gauto2.forstudy.spase/api/users\")\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\requests\\sessions.py\", line 671, in delete\n    return self.request(\"DELETE\", url, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\requests\\sessions.py\", line 589, in request\n    resp = self.send(prep, **send_kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\requests\\sessions.py\", line 703, in send\n    r = adapter.send(request, **kwargs)\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Ирина\\PycharmProjects\\pythonProject\\venv\\Lib\\site-packages\\requests\\adapters.py\", line 519, in send\n    raise ConnectionError(e, request=request)\n"}, "start": 1696247317688, "stop": 1696247319879}], "start": 1696247287527, "stop": 1696247319879}